mava/systems/ppo/ff_mappo.py

```diff
@@ -333,8 +333,8 @@ def learner_setup(
     n_devices = len(jax.devices())
 
     # Get number of actions and agents.
-    num_actions = int(env.action_spec().num_values[0])
-    num_agents = env.action_spec().shape[0]
+    num_actions = env.num_actions
```

Comment:
Can we call this `act_dim` rather? Both in the env and here, as I think this works better for continuous/discrete systems. `num_actions` only really explains discrete